### PR TITLE
Awilix fix

### PR DESCRIPTION
**Pull request by EdwardDrapkin**
There's a bug in how I was defining a few things in this awilix libdef that I've since fixed.  Thanks for your time!

### CODE REVIEW

**Comment by EdwardDrapkin** [comment]
Whoops, my bad.  Good news though, the fix is in! (see what I did there)

**Comment by EdwardDrapkin** [comment]
Sorry about those last two commits, I guess Flow doesn't seem to capture strings as literals when they're parameterized. :(

So it looks like there's no way to reliably model this library completely.

container.register('foo', asValue(123)): flow sees "string" and not "'foo'" :(

File: definitions/npm/awilix_v2.x.x/flow_v0.56.x-/awilix_v2.x.x.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 declare module "awilix" {
-  declare module.exports: {
+  declare module .exports: {
```
Comment (GAntoine) [inline definitions/npm/awilix_v2.x.x/flow_v0.56.x-/awilix_v2.x.x.js]: Pretty sure this is a typo

Comment (EdwardDrapkin) [reply definitions/npm/awilix_v2.x.x/flow_v0.56.x-/awilix_v2.x.x.js]: Jetbrains does that automatically and they haven't fixed it yet, edited and pushed, sorry

**Pull request merged**